### PR TITLE
Improve documentation for std.digest.crc.crc32Of

### PR DESCRIPTION
**Pull request by WalterBright**
And added examples.

### CODE REVIEW

**Comment by davideagen** [comment]
Looks good to me.

File: std/digest/crc.d [hunk 1]
```diff
@@ -308,13 +308,37 @@ unittest
...
  * CRC32 implementation.
+ *
+ * Params:
+ *      data = $(D InputRange) of $(D ElementType) implicitly convertible to $(D ubyte), $(D ubyte[]) or $(D ubyte[num]
+ *             or one or more arrays of any type.
+ *
+ * Returns:
+ *      CRC32 of data
  */
 //simple alias doesn't work here, hope this gets inlined...
```
Comment (yebblies) [inline std/digest/crc.d]: Can this be an alias now?

**Comment by yebblies** [comment]
Auto-merge toggled on

File: std/digest/crc.d (hunk 1 above)
Comment (WalterBright) [inline std/digest/crc.d]: no, because it adds arguments

Comment (yebblies) [inline std/digest/crc.d]: Is that a problem?
`alias crc32Of(T...) = digent!(CRC32, T);`

**Pull request merged**